### PR TITLE
Trending movies UI

### PR DESCRIPTION
**Pull request by LeonardoMolleker**
ezgif com-gif-maker (4)

A grid view of card that contains an image with a loader as a placeholder.

### CODE REVIEW

File: lib/widgets/home_appbar_content.dart [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: [
+
+      ],
+    );
+  }
+}
```
Comment (FranciscoGaraventa) [inline lib/widgets/home_appbar_content.dart]: If you don't use this class, remove it please

Comment (LeonardoMolleker) [reply lib/widgets/home_appbar_content.dart]: Done!

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

**Review by FranciscoGaraventa — APPROVED** [review]
LGTM

**Pull request merged**